Makefile

```diff
@@ -135,8 +135,13 @@ export PRINT_HELP_PYSCRIPT
 err_if_empty = $(if $(strip $($(1))),$(strip $($(1))),$(error Required variable $(1) value is undefined, whitespace, or empty))
 
 .PHONY: help
+ifneq (, ${PYTHON})
```

Comment:
Does this work if Python isn't installed?  I'm getting:

```
# python3 python|head -n1
python3: can't open file 'python': [Errno 2] No such file or directory
```
and think this will force the else to fire, or am I badly parsing the command Makefile will call?

Reply:
Yes that is the idea.

Reply:
Right, and this is probably my confusion, but I thought on a no python machine this was resolving to `if '' ne 'python3: can't open...'` and then on a machine with python `if '' ne 'pythonv3.1' so the else statement wouldn't be ever entered?

Reply:
@TomSweeneyRedHat where are you seeing that `can't open file`? Every way I've tried to test the `PYTHON` definition:

https://github.com/containers/podman/blob/fffcc25d8dfab8c6059c229177286f462d909b8d/Makefile#L33

...gives the right result. `command -v` on bash and even `/bin/sh` on fedora and alpine returns empty string if neither python3 nor python are installed. Can you give more context on the failure you're seeing?

Reply:
Correct, here is an example, added the "<" and ">" to help you visulize
```
echo "<$(command -v python4 | head -n1)>"
<>
```

Reply:
@edsantiago I may have shot myself in the foot, I'd dropped the 'command -v` part of the command to get that error.
@rhatdan thanks for the clarification.